cmd/vsphere-cloud-controller-manager/main.go

```diff
@@ -71,8 +75,12 @@ func main() {
 
 	//
 	var versionFlag *pflag.Value
+	var clusterNameFlag *pflag.Value
 	pflag.CommandLine.VisitAll(func(flag *pflag.Flag) {
-		if flag.Name == "version" {
+		switch flag.Name {
+		case "cluster-name":
```

Comment:
nit: I actually prefer this to be `cluster-id` to match with the cluster ID flag in the CSI driver

Reply:
this is a predefined flag from Kubernetes and cannot be changed.
see https://github.com/kubernetes/kubernetes/blob/2652e176e0a24ed9a5d90cc9b3d670c0d4ed8720/cmd/controller-manager/app/options/kubecloudshared.go#L60

Reply:
I prefer we don't use  this flag, it has some implications that aren't so obvious (the flag description is already telling). 

I think what would be best is adding a `cluster-id` field to the global config https://github.com/kubernetes/cloud-provider-vsphere/blob/master/pkg/common/config/types.go#L21 and then using that value for "cluster name" for LBs.

Reply:
introduced new flag cluster-id

Reply:
The reference docu for the cloud-controller maneger (https://kubernetes.io/docs/reference/command-line-tools-reference/cloud-controller-manager/) explains the option `cluster-name`with 
> --cluster-name string     Default: "kubernetes"
>   The instance prefix for the cluster.
which is exactly the use case we want to use. Therefore it really seems to to be the right choice. It also already passed by the load balancer interface of the cloud controller manager. That's another clear indicator that we are using it as intended.